mod/discussions/views/default/css/discussion.php

```diff
@@ -0,0 +1,4 @@
+
+.discussions-latest-reply {
```

Comment:
Maybe make the view parallel the CSS class name: like css/elgg/discussions.css (if you go with elgg-discussions-latest-reply, otherwise css/discussions.css if you go with discussions-latest-reply).